model/Export/QTIPackedItemExporter.php

```diff
@@ -199,10 +199,13 @@ public function exportManifest($options = [], $exportResult = [])
             }
 
             $manifest = $this->getManifest();
-            $this->getMetadataExporter()->export($this->getItem(), $manifest);
-            $this->injectMetadataToManifest($manifest, $this->getItem());
-            $this->setManifest($manifest);
+            // APIP Package could not include metadata
+            if (!(isset($options['apip']) && $options['apip'] === true)) {
```

Comment:
Can we use some variable here? It will greatly increase readability.  `$isNotApip = !(isset($options['apip']) && $options['apip'] === true)`